Not sure about the linter error: ` > pwnlib/commandline/template.py:63:8: E0611: No name 'TerminalFormatter' in module 'pygments.formatters' (no-name-in-module)`
Maybe we should shut up pylint as Microsoft did https://github.com/microsoft/knack/blob/50d30cfc3477aff34451231359704e961e9abd3f/knack/output.py#L43:

```python
return highlight(format_json(obj), lexers.JsonLexer(), formatters.TerminalFormatter())  # pylint: disable=no-member
```

Where would I add the issues in the CHANGELOG.md?

Thanks :)

--- a/pwnlib/commandline/template.py
+++ b/pwnlib/commandline/template.py
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python2
+from __future__ import absolute_import
+from __future__ import division
 
 from pwn import *
 from pwnlib.commandline import common

